x-pack/plugins/apm/public/components/app/Settings/AgentConfigurations/AgentConfigurationCreateEdit/SettingsPage/SettingsPage.tsx

```diff
@@ -136,79 +136,65 @@ export function SettingsPage({
-                  titleSize="xs"
-                  title={
-                    isLoading
-                      ? '-'
-                      : getOptionLabel(newConfig.service.environment)
-                  }
-                  description={i18n.translate(
-                    'xpack.apm.agentConfig.chooseService.service.environment.label',
-                    { defaultMessage: 'Environment' }
-                  )}
-                />
-              </EuiFlexItem>
-              <EuiFlexItem grow={false}>
-                {!isEditMode && (
-                  <EuiButton onClick={onClickEdit} iconType="pencil">
-                    {i18n.translate(
-                      'xpack.apm.agentConfig.chooseService.editButton',
-                      { defaultMessage: 'Edit' }
-                    )}
-                  </EuiButton>
+          <EuiFlexGroup>
+            <EuiFlexItem>
+              <EuiStat
+                titleSize="xs"
+                title={isLoading ? '-' : getOptionLabel(newConfig.service.name)}
+                description={i18n.translate(
+                  'xpack.apm.agentConfig.chooseService.service.name.label',
+                  { defaultMessage: 'Service name' }
                 )}
-              </EuiFlexItem>
-            </EuiFlexGroup>
-          </EuiPanel>
+              />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiStat
+                titleSize="xs"
+                title={
+                  isLoading
+                    ? '-'
+                    : getOptionLabel(newConfig.service.environment)
+                }
+                description={i18n.translate(
+                  'xpack.apm.agentConfig.chooseService.service.environment.label',
+                  { defaultMessage: 'Environment' }
+                )}
+              />
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              {!isEditMode && (
+                <EuiButton onClick={onClickEdit} iconType="pencil">
+                  {i18n.translate(
+                    'xpack.apm.agentConfig.chooseService.editButton',
+                    { defaultMessage: 'Edit' }
+                  )}
+                </EuiButton>
+              )}
+            </EuiFlexItem>
+          </EuiFlexGroup>
 
-          <EuiSpacer size="m" />
+          <EuiHorizontalRule />
 
           {/* Settings panel */}
-          <EuiPanel paddingSize="m">
-            <EuiTitle size="s">
-              <h3>
-                {i18n.translate('xpack.apm.agentConfig.settings.title', {
-                  defaultMessage: 'Configuration options',
-                })}
-              </h3>
-            </EuiTitle>
 
-            <EuiSpacer size="m" />
+          <EuiTitle size="s">
+            <h3>
+              {i18n.translate('xpack.apm.agentConfig.settings.title', {
+                defaultMessage: 'Configuration options',
+              })}
+            </h3>
+          </EuiTitle>
+
+          <EuiSpacer size="m" />
```

Comment:
I think this unnecessary in this new layout - it's pretty clear that the following form fields are configuration options so the title is just redundant. We used to have this in two panels, but this is much nicer.

<img width="1301" alt="Screenshot 2021-06-09 at 09 48 23" src="https://user-images.githubusercontent.com/4104278/121315377-93b58400-c908-11eb-909b-e4561acf971d.png">

Reply:
I agree. Will remove